feat: Attendance Status and some enhancements (#179)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/human-resources/attendance-status.md

<!-- add-breadcrumbs -->
# Attendance Status

Attendance Status the status of an employee, Which represent That wether Employee is Present, Absent, or is on leave on particular day.

In ERPNext You can create multiple Status for Attendance. Suppose, Employee is on visit to other place or working remotely in that case we allow you to create custom status for tracking Attendance.

To access Attendance Status, go to:

> > Home > Human Resources > Attendance

Author: It will be attendance status. Because I have created a master for attendance status.

## 1. How to Create Attendance status:

1. Go to the Attendance Status list, click on New.
1. Name your Attendance Status.
1. Write Abbreviation for your status. Which is used in Monthly Attendance Sheet.

> 1. Click on save.

Reviewer: ```suggestion
1. Click on Save.
```

> > Note: if None of the property Is Half Day, Is Leave or Is Present is checked status will be treated as Absent

Reviewer: ```suggestion
> Note: If none of the properties, i.e. "Is Half Day", "Is Leave" or "Is Present" is checked, then the status will be treated as Absent.
```

> <img class="screenshot" alt="Attendance" src="{{docs_base_url}}/assets/img/human-resources/attendance-status.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Attendance Status" src="{{docs_base_url}}/assets/img/human-resources/attendance-status.png">
```

> Below are detailed explanation of all checkbox in Attendance Status.

Reviewer: ```suggestion
Below is a detailed explanation of all the checkboxes in Attendance Status.
```

> * **IS Half Day:** If Checked, The Attendance Status will represent Half Day Status During Attendance Creation.

Reviewer: ```suggestion
* **Is Half Day:** If Checked, the Attendance Status will be set to Half Day during the attendance creation.
```

Author: There is nothing to be set. Attendance Status is a doctype where users can define their own attendance status.

> * **IS Leave:** In ensure that Attendance Status will treated as leave Status.

Reviewer: ```suggestion
* **Is Leave:** If checked, the Attendance Status will be set as Leave during the attendance creation.
```

Author: There is nothing to be set. Attendance Status is a doctype where users can define their own attendance status.

> * **Is Present:** If Checked, The Attendance Status will be treated as present.

Reviewer: ```suggestion
* **Is Present:** If checked, the Attendance Status will be set as Present.
```

Author: There is nothing to be set. Attendance Status is a doctype where users can define their own attendance status.

> * **Applicable for Leave Application**: This checkbox is visible only if **Is Half Day** or **Is leave** property for Attendance Status is enable. If this property is enabled then it will be taken as default Leave or Half Day Status for Attendance created while Leave approval.

Reviewer: ```suggestion
* **Applicable for Leave Application**: This checkbox is visible only if the **Is Half Day** or **Is Leave** property for Attendance Status is enabled. 
```

Reviewer: If this property is enabled then it will be taken as default Leave or Half Day Status for Attendance created while Leave approval. 

what does this mean?

> >Note: Only one Half Day and Leave Status with property Applicable for Leave Application is allowed at a time.

Reviewer: ```suggestion
> Note: Only one "Half Day" and Leave Status with property, "Applicable for Leave Application" is allowed at a time.
```

> * **Applicable for Employee Checkins**: This Property works for Auto Attendance Creation through Employee Checkin. If Checked the status will be taken as default status for Present, Half day, or Leave during automatic attendance creation.

Reviewer: ```suggestion
* **Applicable for Employee Checkins**: This property works for Auto Attendance Creation through Employee Checkin. If checked, the status will be taken as a default status for "Present", "Half Day", or "Leave" during the automatic attendance creation.
```

> >Note:

Reviewer: ```suggestion

```

> It is mandatory to create the Attendance Status wil this property to make Auto Attendance Functional.

Reviewer: what does this mean?

> Only one Half day, Present, and Absent with property Applicable for Employee Checkins ia allowed at a time.

Reviewer: delete this line...this is repeated

> * **Applicable for Attendance Request:** This checkbox is visible if **Is Half Day** for Attendance Status is enable. If Checked, Status will Be treated as default Half day status while Attendance Request Approval.

Reviewer: ```suggestion
* **Applicable for Attendance Request:** This checkbox is visible if **Is Half Day** for Attendance Status is enabled. If checked, the status will be set to Half Day during the Attendance Request approval.
```
